playground/worker/__tests__/relative-base/relative-base-worker.spec.ts

```diff
@@ -14,13 +14,19 @@ test('normal', async () => {
     'worker bundle with plugin success!',
     true
   )
+  await untilUpdated(
+    () => page.textContent('.asset-url'),
+    isBuild ? '/other-assets/vite' : '/vite.svg',
+    true
+  )
 })
 
 test('TS output', async () => {
   await untilUpdated(() => page.textContent('.pong-ts-output'), 'pong', true)
 })
 
-test('inlined', async () => {
+// TODO: inline worker should inline assets
+test.skip('inlined', async () => {
```

Comment:
Inside inline worker, `self.location.href` and `import.meta.url` are both `blob:foobar` or `data:foobar`. So `new URL('./foo', import.meta.url)` will become invalid URL.
Do we need to inline assets for inline workers? Or do we need to show an error for them? Or pass the current url into them?

Reply:
Can we treat inline workers like lib mode(should we make all the asset to base64)?

Reply:
I think we can. But I'm not sure how this should be handled.

Also this will need much more work and since this wasn't working before this PR, I think we shouldn't include the change in this PR.